src/common/components/generics/MenuItem.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
You can group these resource under `/generics/Sidebar`

Reply:
Grouping finish